utils/string_utils.js

```diff
@@ -1,21 +1,5 @@
-const wordsStandardize = (word) => {
-	if (word) {
-		const wordStandard = word.trim().toLowerCase();
-		return wordStandard.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
-	}
-	return 'null';
-};
-
-const wordToBoolean = word => {
-	if (word === '') return true;
-	if (word) return word.toLowerCase() === 'true' || word === '1';
-	return false;
-};
-
-const splitQuery = (query) => query.indexOf('|') === -1 ? query.split(',') : query.split('|');
-
 module.exports = {
-	wordsStandardize,
-	wordToBoolean,
-	splitQuery
+	wordsStandardize: (word) => word ? word.trim().toLowerCase().normalize('NFD').replace(/[\u0300-\u036f]/g, '') : 'null',
+	wordToBoolean: (word) => word === '' || (word ? word.toLowerCase() === 'true' || word === '1' : false),
+	splitQuery: (query) => query.indexOf('|') === -1 ? query.split(',') : query.split('|')
```

Comment:
It's good to shorten these functions, but they should be declared as const and then exported. It's better practice.
```js
wordsStandardize = (word) => { ... };
wordToBoolean = (word) => { ... };
splitQuery = (word) => { ... };
module.exports = {
  wordsStandardize,
  wordToBoolean,
  splitQuery
}